lib/src/solana_client.dart

```diff
@@ -207,11 +207,17 @@ class SolanaClient {
   ///
   /// Providing [before] and [until] also moves the cursor to a more specific
   /// subset.
+  ///
+  /// For [commitment] parameter description see
+  /// https://docs.solana.com/developing/clients/jsonrpc-api#configuring-state-commitment
+  ///
+  /// [TxStatus.processed] is not supported as [commitment].
```

Comment:
What does this comment mean?

Reply:
https://docs.solana.com/developing/clients/jsonrpc-api#getconfirmedtransaction

> (optional) Commitment; "processed" is not supported. If parameter not provided, the default is "finalized".

Reply:
Ok, understood.